protocol/messenger.go

```diff
@@ -4156,13 +4155,13 @@ func (m *Messenger) MessageByChatID(chatID, cursor string, limit int) ([]*common
 }
 
 func (m *Messenger) prepareMessages(messages map[string]*common.Message) error {
-	if m.httpServer != nil {
-		for idx := range messages {
-			err := m.prepareMessage(messages[idx], m.httpServer)
-
-			if err != nil {
-				return err
-			}
+	if m.httpServer == nil {
+		return nil
+	}
+	for idx := range messages {
```

Comment:
Isn't this the same as :

```go
for _, message := range messages {
    if err := m.prepareMessage(message, m.httpServer); err != nil {
        return err
    }
}
```

Reply:
Yep, it is. But the main idea of this fix was to quick return on `m.httpServer == nil`. Didn't want to make too many changes in the RC fix. But this one I just couldn't resist 😄